dot/sync/chain_sync_integration_test.go

```diff
@@ -574,9 +575,10 @@ func TestChainSync_doSync_Integration(t *testing.T) {
 
 	// test to see if descending blocks get reversed
 	req.Direction = network.Descending
-	cs.network = new(mocks.Network)
-	cs.network.(*mocks.Network).On("DoBlockRequest", mock.AnythingOfType("peer.ID"),
-		mock.AnythingOfType("*network.BlockRequestMessage")).Return(resp, nil)
+	net = NewMockNetwork(ctrl)
+	net.EXPECT().DoBlockRequest(gomock.AssignableToTypeOf(peer.ID("0")),
+		gomock.AssignableToTypeOf(&network.BlockRequestMessage{})).Return(resp, nil).AnyTimes()
+	cs.network = net
```

Comment:
1. Can the arguments be easily predictable? If so let's have them here directly
2. Try to find the number of times each get called.